sources/sources.go

```diff
@@ -56,10 +56,17 @@ func init() {
 		Feed:      httpProxyRegexFeed("https://free-proxy-list.net/uk-proxy.html", "UK Proxy List"),
 	}, Source{
 		ID:        55,
-		Homepage:  "https://proxy-list.download/",
+		Homepage:  "https://www.proxy-list.download/",
 		Frequency: 3 * time.Hour,
 		Seed:      true,
 		Feed:      proxyListDownload,
+	}, Source{
```

Comment:
please do it in `regex.go`, that i've introduced in https://github.com/nfx/slrp/pull/47

example: https://github.com/nfx/slrp/blob/main/sources/regex.go#L25-L30